crypto/aes/aes_ige.c

```diff
@@ -50,7 +50,7 @@ void AES_ige_encrypt(const unsigned char *in, unsigned char *out,
                      unsigned char *ivec, const int enc)
 {
     size_t n;
-    size_t len = length;
+    size_t len;
```

Comment:
I am concerned about removing assignments in initializers, like this and by_dir.c line 231.  The project has the habit of initializing things at declaration, to avoid "maybe uninitialized" errors, and removing these two seems like going too far.  Especially since these are, as you point out, false positives.

Reply:
IMO this can be removed. There is an initializer only a few lines below. Alternatively the initializer down there should be removed and this one replaced with  `size_t len = length / AES_BLOCK_SIZE;`